src/components/AddRepsModal/style.css

```diff
@@ -0,0 +1,15 @@
+.set-rep-input {
+  width: 40px !important;
```

Comment:
Isn't `!important` a CSS antipattern?

Reply:
Yes, it is. I thought it was the only way to override bootstrap classes, but I found a better one, will fix it without `important`.